Entities/Attachable.cpp

```diff
@@ -112,10 +115,12 @@ int Attachable::ReadProperty(std::string propName, Reader &reader)
         m_pBreakWound = dynamic_cast<const AEmitter *>(g_PresetMan.GetEntityPreset(reader));
     else if (propName == "JointOffset")
         reader >> m_JointOffset;
-	else if (propName == "InheritsRotAngle")
-		reader >> m_InheritsRotAngle;
-	else if (propName == "DrawAfterParent")
+    else if (propName == "InheritsRotAngle")
+        reader >> m_InheritsRotAngle;
+    else if (propName == "DrawAfterParent")
         reader >> m_DrawAfterParent;
+    else if (propName == "CollidesWithTerrain")
```

Comment:
Does m_CollidesWithTerrain not get set from ini as well? Do you have to set CollidesWithTerrain = true in ini and then call EnableCollisions in lua for it to work, or am I missing something?

Reply:
No, m_CollidesWithTerrain does not get set in ini.
m_IsSetToCollide is the one getting set, but the ini property is called CollidesWithTerrain.
If CollidesWithTerrain = 1 in ini, EnableTerrainCollisions(true) will be called in cpp, enabling collisions.
You do not need to call it through lua, but you are now able to.
If CollidesWithTerrain = 0 in ini, EnableTerrainCollisions in cpp will not be called at all, and calling it through lua will spit out an error.

Reply:
Okay, so there's something that makes it call EnableTerrainCollisions(true) when CollidesWithTerrain = 1 is set in ini? I guess in Create somewhere?